src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -411,7 +412,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     completionWork onFailure {
       case e =>
         logger.error(s"Preemption work failed for call ${callKey.tag}! " + e.getMessage, e)
-        scheduleTransition(WorkflowFailed)
+        self ! CheckForWorkflowComplete
```

Comment:
Yes there are surely a few other places where it's needed, but pretty much everytime we call persistStatusAndAck there's no need to send `CheckForWorkflowComplete` just after IMO